src/components/30-organisms/modal/package.json

```diff
@@ -26,6 +26,7 @@
     "url": "https://github.com/axa-ch/patterns-library/issues"
   },
   "dependencies": {
+    "@axa-ch/materials": "14.0.0",
```

Comment:
Without the `^` it seems that lerna do not update this version. But lerna updates it anyway. Also to breaking change versions.